skfuzzy/control/visualization.py

```diff
@@ -48,21 +56,27 @@ def view(self, *args, **kwargs):
                     self.fuzzy_var.universe, zeros, cut_mfs[label],
                     facecolor=color, alpha=0.4)
 
-        # Plot defuzzified output if available
-        if len(cut_mfs) > 0:
-            crip_value = defuzz(self.fuzzy_var.universe, output_mf,
-                                self.fuzzy_var.defuzzify_method)
-            if crip_value is not None:
+        # Plot crisp value if available
+        if len(cut_mfs) > 0 and not all(output_mf == 0):
+            crisp_value = None
+            if hasattr(self.fuzzy_var, 'input'):
+                crisp_value = self.fuzzy_var.input[sim]
+            elif hasattr(self.fuzzy_var, 'output'):
+                crisp_value = self.fuzzy_var.output[sim]
+
+            if crisp_value is not None:
                 y = interp_membership(self.fuzzy_var.universe,
-                                      output_mf, crip_value)
-                self.ax.plot([crip_value] * 2, [0, y],
+                                      output_mf, crisp_value)
+                if y < 0.1:
+                    y = 1.
+                self.ax.plot([crisp_value] * 2, [0, y],
                               color='k', lw=3, label='crisp value')
 
         return self.fig
 
     def _init_plot(self):
         # Formatting: limits
-        self.ax.set_ylim([0, 1])
+        self.ax.set_ylim([0, 1.01])
```

Comment:
I'm already turnning off the axis on line 96.  This was done because if a membership is 1.0 it doesn't show up well.  In your examples, it looks fine because you're using triangle membership functions and they're 1.0 for a single point and thus you don't really notice it.  But when a mf is at 1.0 for a long time, I was noticing it was hard to see.